cypress/integration/menu/menu.js

```diff
@@ -1,16 +1,21 @@
-import { Given, When, Then } from "cypress-cucumber-preprocessor/steps"
+import { Given, When, Then, Before } from "cypress-cucumber-preprocessor/steps"
 import { v4 as uuidv4 } from 'uuid';
 
-const url = 'https://frontend.sandbox.nva.aws.unit.no';
+const USER_NAME = 'testUser@unit.no';
+
+Before(async () => {
+  cy.addUser(USER_NAME).as('authResult');
+});
 
 Given('that the user is not logged in', () => {
-    cy.visit(url);
-    cy.get('[data-testid=menu-login-button]').should('be.visible');
+  cy.visit('/');
+  cy.get('[data-testid=menu-login-button]').should('be.visible');

```

Comment:
denne skal kanskje ikke være her, siden samme sjekk gjøres i `Then('they see the Log in button', (...)`?

Reply:
To forskjellige sider som sjekkes.